homework/index.js

```diff
@@ -16,6 +17,11 @@
     xhr.send();
   }
 
+  function renderError(error) {
+    const root = document.getElementById('root');
+    createAndAppend('h1', root, { text: error.message });
```

Comment:
You have an ESLint error because you are referencing the `createAndAppend` function before it is defined. The solution is simple: just move this function definition after that of  `createAndAppend`.